resources/config-defaults.edn

```diff
@@ -21,12 +21,6 @@
         :search-root "dc=Suivohtor,dc=local"}
  :alternative-login-url nil
 
- ;; Open ID Connect settings
- ; These allow login to http://localhost:3000/oidc-callback only
- :oidc-domain "remsdev.eu.auth0.com"
- :oidc-client-id "65TIN1Joo6EvsHSPCN4zyCaxxPh0WJF1"
- :oidc-client-secret "e1yr9R_8WW3Xg03wm6wJ4UmNVVjiqdVhwFGvHhttyoRS3h04F4MGh0zRaV7vKm30"
-
```

Comment:
The `config-defaults.edn` should have at least `nil` placeholders for all the configuration parameters. It's linked as a configuration reference in https://github.com/CSCfi/rems/blob/master/docs/configuration.md